src/libraries/System.Drawing.Common/tests/GraphicsTests.cs

```diff
@@ -12,6 +12,8 @@ namespace System.Drawing.Tests
 {
     public partial class GraphicsTests
     {
+        public bool IsWindows7OrWindowsArm64 => PlatformDetection.IsWindows7 || (PlatformDetection.IsWindows && PlatformDetection.IsArm64);
```

Comment:
This doesn't need to be static?

Reply:
Yes it needs to be static: https://github.com/dotnet/arcade/blob/82b37b7f7a37ae812b028519fe2c1835e5dc6d5d/src/Microsoft.DotNet.XUnitExtensions/src/Discoverers/ConditionalTestDiscoverer.cs#L135. Thanks, fixed.